test/user-agent/karma.conf.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (config) {
             }
         },
         client: {
-            captureConsole: false,
+            captureConsole: true,
```

Comment:
you shouldn't enable console, revert this. That's for debug purpose only. 

Otherwise bddtest steps will get lost between console output.